src/components/schemas/expense_policies.yaml

```diff
@@ -0,0 +1,507 @@
+policy_in:
+  type: object
+  required:
+    - source
+  additionalProperties: False
+  properties:
+    id:
+      allOf:
+        - $ref: ./fields.yaml#/id_string
+      description: |
+        This represents the id of the policy.
+      example: 'tpr12345xyz'
+    description:
+      allOf:
+        - $ref: ./fields.yaml#/description
+      description: |
+        This represents the description of the policy.
+      example: Receipt is mandatory for expense above $50
+    source:
+      $ref: ./fields.yaml#/policy_source
+    expense_limit_type:
+      $ref: ./fields.yaml#/policy_limit_type
+    expense_currency:
+      $ref: ./fields.yaml#/currency
+    expense_amount_max:
+      allOf:
+        - $ref: ./fields.yaml#/amount
+      example: 10000
+    expense_amount_min:
+      allOf:
+        - $ref: ./fields.yaml#/amount
+      example: 500
+    expense_distance_max:
+      allOf:
+        - $ref: ./fields.yaml#/distance
+      example: 10000
+    expense_distance_min:
+      allOf:
+        - $ref: ./fields.yaml#/distance
+      example: 500
+    expense_category_ids:
+      type: array
+      items:
+        $ref: './fields.yaml#/id_integer'
+      example: [2222, 1221]
+      description: |
+          List of category ids
```

Comment:
more elaborate on how this policy will be used for expenses belonging to this category